README.md

```diff
@@ -113,6 +113,10 @@ GCS bucket in the project in `deploy.json`, even for local deployment of Data
 Explorer. Choose a project where you have at least Project Editor permissions.
 * TODO(melissachang): Add instructions for creating bucket in the project
 specified in `deploy.json`.
+* The Export to Saturn feature creates a signed URL, which requires a service
+account private key. Follow these instructions (["Generate a new private key"](https://cloud.google.com/storage/docs/access-control/create-signed-urls-program#signing-language))
+to create the private key. Select the App Engine default service account.
+Store the key in `api/data_explorer/private-key.json`.
```

Comment:
Ideally we would not export keys for service accounts because it reduces the level of security on those accounts. Is there a reason you can't just use Application Default Credentials?

Reply:
Per offline discussion, this seems fine as long as we're careful with the keys.